query/query.md

```diff
@@ -0,0 +1,72 @@
+Queries return all documents that match the user supplied key-pairs. This includes document attestation details (`signers` and `location`). Data is retrieved from the [Chainvine Expanse API](https://hub.docker.com/r/slafazan/chainvine-expanse).
+
+## Example Query
+
+- What is the quantity of graphite being shipped?
+
+Return all documents of a certain type (`Purchase Order`) involving the correct entities.
+
+```json
```

Comment:
Thanks for including this documentation. Is this targeting the documents endpoint/function?

Reply:
it hits multiple endpoints on the chainvine API to return an array of documents with entity, attestation and location details